internal/controllers/router/tests/create-full/01-assert.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Router
+metadata:
+  name: create-full
+status:
+  resource: 
```

Comment:
We should also check for the neutron metadata. This should now be possible via CEL validations like I did in https://github.com/k-orc/openstack-resource-controller/blob/main/internal/controllers/image/tests/create-full/00-assert.yaml for example.

Note that this depends on https://github.com/k-orc/openstack-resource-controller/pull/287.